resources/views/root/root_home.blade.php

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
__[code_sniffer]__
* No PHP code was found in this file and short open tags are not allowed by this install of PHP. This file may be using short open tags but PHP does not allow them.